src/transferlistwidget.cpp

```diff
@@ -741,6 +754,10 @@ void TransferListWidget::displayListMenu(const QPoint&)
     connect(&actionForce_recheck, SIGNAL(triggered()), this, SLOT(recheckSelectedTorrents()));
     QAction actionCopy_magnet_link(QIcon(":/Icons/magnet.png"), tr("Copy magnet link"), 0);
     connect(&actionCopy_magnet_link, SIGNAL(triggered()), this, SLOT(copySelectedMagnetURIs()));
+
+    QAction actionCopy_caption_list(tr("Copy name"), 0);
```

Comment:
Add the copy icon:
```c++
QAction actionCopy_caption_list(IconProvider::instance()->getIcon("edit-copy"), tr("Copy name"), 0);
```
And I think is better to put this option before "Copy magnet". "Copy magnet" the last one.